gensim/models/wrappers/ldamallet.py

```diff
@@ -190,7 +190,7 @@ def load_word_topics(self):
         if hasattr(self.id2word, 'token2id'):
             word2id = self.id2word.token2id
         else:
-            word2id = dict((v, k) for k, v in iteritems(self.id2word))
+            word2id = dict((v, k) for k, v in iteritems(dict(self.id2word)))
```

Comment:
It's for python3 compatibility (if self.id2word is `FakeDict` object, `iteritems` can't work correctly)